group_vars/all/osa.yml

```diff
@@ -58,6 +58,15 @@ nova_console_type: novnc
 # RabbitMQ overrides
 rabbitmq_ulimit: 65535
 
+rabbitmq_policies:
+  - name: "HA"
+    pattern: '^(?!amq\.).*'
+    tags:
+      ha-mode: exactly
+      ha-params: 2
```

Comment:
Can this be changed to a variable based off what the quorum is.  2 in a 3 node cluster 3 in a 5 node, etc.  That'd match the rabbit docs of this.  Other than that, lgtm.

https://www.rabbitmq.com/ha.html